src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+
+    public String getRandomBall() {
+        Ball ball = new Ball(getRandomColor());
+        Random random = new Random();
```

Comment:
Make random and colourSupplier variables (which you will create) fields of the class, as you did in the ColorSupplier class